modular_skyrat/modules/blueshield/code/modsuit/mod_theme.dm

```diff
@@ -15,7 +15,7 @@
 	atom_flags = PREVENT_CONTENTS_EXPLOSION_1
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	siemens_coefficient = 0
-	complexity_max = DEFAULT_MAX_COMPLEXITY + 3
+	complexity_max = DEFAULT_MAX_COMPLEXITY // Bubberstation edit: original 18
```

Comment:
```suggestion
	complexity_max = DEFAULT_MAX_COMPLEXITY // Bubberstation edit, original: DEFAULT_MAX_COMPLEXITY + 3
```

Reply:
Also can be made modular

Reply:
> Also can be made modular

I'll refactor this if I make more modsuit overrides then just the blueshield